action.yml

```diff
@@ -14,3 +14,6 @@ outputs:
 runs:
   using: docker
   image: docker://ghcr.io/readmeio/rdme:8.6.0
+  args:
+    - docker-gha
```

Comment:
This is a new internal argument we send to indicate to the `rdme` Docker image that the command is going to be a single string as opposed to an array of strings.

Reply:
This is a pretty neat/sufficient solution. Annoying that it's necessary, but it's not too bad.